AWSS3/AWSS3TransferUtility.h

```diff
@@ -16,7 +16,7 @@
 #import <UIKit/UIKit.h>
 #import <AWSCore/AWSCore.h>
 
-FOUNDATION_EXPORT NSString *const AWSS3SessionInvalidatedNotification;
+FOUNDATION_EXPORT NSString *const AWSS3TransferUtilityURLSessionDidBecomeInvalidNotification;
```

Comment:
One very small change recommendation. This line should go below `NS_ASSUME_NONNULL_BEGIN` so that Xcode won't complain about the nullability annotation. Thank you so much for the pull request. We'll merge it in the next release. Thanks.